### PR TITLE
null state improvements

### PR DESCRIPTION
**Pull request by placegraphichere**
- hover, select states for recent/new files
- removed section border on last child
- added column for exiting Ds from null state

### CODE REVIEW

File: src/js/jsx/PanelSet.jsx [hunk 1]
```diff
@@ -279,10 +279,21 @@ define(function (require, exports, module) {
                 );
             } else if (this.state.recentFilesInitialized) {
                 return (
-                    <div ref="panelSet" className="panel-set__container">
-                        <div className="panel panel__visible">
+                    <div className="panel-set panel-set__active null-state">
+                        <PanelColumn
```
Comment (mcilroyc) [inline src/js/jsx/PanelSet.jsx]: The neighboring elements, when they have only one attribute, are kept to one line.. so maybe do that here

File: src/js/jsx/PanelSet.jsx [hunk 2]
```diff
@@ -279,10 +279,21 @@ define(function (require, exports, module) {
...
+                            visible="true">
                             <RecentFiles recentFiles={this.state.recentFiles} />
                             <ArtboardPresets />
+                        </PanelColumn>
+                        <div className="panel__tab-bar panel__tab-bar_visible">
```
Comment (mcilroyc) [inline src/js/jsx/PanelSet.jsx]: double underscore before visible

**Comment by mcilroyc** [comment]
two nits - otherwise :white_check_mark:

**Comment by placegraphichere** [comment]
back atcha

**Pull request merged**